nibabel/cifti/__init__.py

```diff
@@ -0,0 +1,28 @@
+# emacs: -*- mode: python-mode; py-indent-offset: 4; indent-tabs-mode: nil -*-
+# vi: set ft=python sts=4 ts=4 sw=4 et:
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+#
+#   See COPYING file distributed along with the NiBabel package for the
+#   copyright and license terms.
+#
+### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ### ##
+"""CIfTI format IO
+
+.. currentmodule:: nibabel.cifti
+
+.. autosummary::
+   :toctree: ../generated
+
+   ciftiio
+   cifti
+"""
+
+from .parse_cifti_fast import create_cifti_image
+from .cifti import (CiftiMetaData, CiftiHeader, CiftiImage, CiftiLabel,
```

Comment:
I agree with @matthew-brett; I'd rather this be called `Cifti2Image`. If that's agreed, would all of these objects be called `Cifti2*`, or are they the same as for the `Cifti1Image` spec? If not sure, I'd rather they all be renamed to `Cifti2*`, just to be careful and explicit.